tests/consensus_tests/test_partial_snapshot.py

```
@@ -0,0 +1,173 @@
+
+    config = {
+        "QDRANT__LOG_LEVEL": "debug,collection::common::file_utils=trace",
+        "QDRANT__FEATURE_FLAGS__USE_MUTABLE_ID_TRACKER_WITHOUT_ROCKSDB": "true",
+    }
+
+    uris, _, _ = start_cluster(path, 1, port_seed = port, extra_env = config)
+
+    return uris[0]
+
+def bootstrap_collection(peer_url, bootstrap_points = 0):
+    create_collection(peer_url, shard_number = 1, replication_factor = 1, indexing_threshold = 1000000, sparse_vectors = False)
```

Ruff (0.8.2)

`create_collection` may be undefined, or defined from star imports

(F405)